addons/web_editor/static/src/js/transcoder.js

```diff
@@ -12,7 +12,24 @@ var getMatchedCSSRules = function (a) {
     if (!rulesCache.length) {
         var sheets = document.styleSheets;
         for(var i = sheets.length-1; i >= 0 ; i--) {
-            var rules = sheets[i].rules || sheets[i].cssRules;
+            var rules;
+            if (sheets[i].rules) {
+                rules = sheets[i].rules;
+            } else {
+                //try...catch because Firefox not able to enumerate document.styleSheets[].cssRules[] for cross-domain stylesheets.
+                try {
+                    if (!sheets[i].cssRules) {
+                        continue;
+                    }
```

Comment:
shorter if you set "rules = sheets[i].cssRules;" here and remove the condition and the line 30